src/py42/services/administration.py

```diff
@@ -2,11 +2,6 @@
 
 
 class AdministrationService(BaseService):
-    def get_diagnostics(self, include_volumes=None):
```

Comment:
not sure if this requires a CHANGELOG entry since it sounds like no one could've been using it anyway ?